db/schema.rb

```diff
@@ -86,349 +90,391 @@
   add_index "user_identifiers", ["user_id"], name: "index_user_identifiers_on_user_id", using: :btree
 
   create_table "users", force: :cascade do |t|
-    t.string   "firstname"
-    t.string   "surname"
-    t.string   "email",                  default: "", null: false
-    t.string   "orcid_id"
-    t.string   "shibboleth_id"
+    t.string   "firstname",              limit: 255
+    t.string   "surname",                limit: 255
+    t.string   "email",                  limit: 255, default: "", null: false
+    t.string   "orcid_id",               limit: 255
+    t.string   "shibboleth_id",          limit: 255
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "encrypted_password",     default: ""
-    t.string   "reset_password_token"
+    t.string   "encrypted_password",     limit: 255, default: ""
+    t.string   "reset_password_token",   limit: 255
     t.datetime "reset_password_sent_at"
     t.datetime "remember_created_at"
-    t.integer  "sign_in_count",          default: 0
+    t.integer  "sign_in_count",          limit: 4,   default: 0
     t.datetime "current_sign_in_at"
     t.datetime "last_sign_in_at"
-    t.string   "current_sign_in_ip"
-    t.string   "last_sign_in_ip"
-    t.string   "confirmation_token"
+    t.string   "current_sign_in_ip",     limit: 255
+    t.string   "last_sign_in_ip",        limit: 255
+    t.string   "confirmation_token",     limit: 255
     t.datetime "confirmed_at"
     t.datetime "confirmation_sent_at"
-    t.string   "invitation_token"
+    t.string   "invitation_token",       limit: 255
     t.datetime "invitation_created_at"
     t.datetime "invitation_sent_at"
     t.datetime "invitation_accepted_at"
-    t.string   "other_organisation"
+    t.string   "other_organisation",     limit: 255
     t.boolean  "accept_terms"
-    t.integer  "org_id"
-    t.string   "api_token"
-    t.integer  "invited_by_id"
-    t.string   "invited_by_type"
-    t.integer  "language_id"
-    t.string   "recovery_email"
+    t.integer  "org_id",                 limit: 4
+    t.string   "api_token",              limit: 255
+    t.integer  "invited_by_id",          limit: 4
+    t.string   "invited_by_type",        limit: 255
+    t.integer  "language_id",            limit: 4
+    t.string   "recovery_email",         limit: 255
   end
 
   add_index "users", ["email"], name: "index_users_on_email", unique: true, using: :btree
+  add_index "users", ["language_id"], name: "fk_rails_45f4f12508", using: :btree
+  add_index "users", ["org_id"], name: "fk_rails_e73753bccb", using: :btree
   add_index "users", ["org_id"], name: "index_users_on_org_id", using: :btree
 
   create_table "users_perms", id: false, force: :cascade do |t|
-    t.integer "user_id"
-    t.integer "perm_id"
+    t.integer "user_id", limit: 4
+    t.integer "perm_id", limit: 4
   end
 
+  add_index "users_perms", ["perm_id"], name: "fk_rails_457217c31c", using: :btree
```

Comment:
Not sure I see a use case for this index.  was this added by this piece of code?
I dont think we ever select from this table users based on a perm_id